src/components/page/PrTable.js

```diff
@@ -39,11 +58,57 @@ const PrTable = ({ query }) => {
     {},
     [query]
   );
+  const [prRecords, setPrRecords] = useState({});
+
+  useEffect(() => {
+    let mounted = true;
+    if (!data) return;
+
+    const refNums = [];
+    const prsByRefNum = {};
+
+    for (let pr of data) {
+      const refNum = pr.title.split(" ")[0].toUpperCase();
+      if (!/[A-Z]+-[0-9]+/.test(refNum)) continue;
```

Comment:
Numbers are valid in the prefix after the first letter.

Reply:
Fixed